docs/Onco-Query-Language.md

```diff
@@ -41,7 +41,7 @@ Data Type | Keywords and Syntax | Default*
 --------- | ------------------- | --------
 Mutations | `MUT` All non-synonymous mutations <br> `MUT = <protein change>` Specific amino acid changes (e.g. `V600E` or `V600`) <br> `MUT = <mutation type>` Acceptable values are: `MISSENSE, NONSENSE, NONSTART, NONSTOP, FRAMESHIFT, INFRAME, SPLICE, TRUNC` | `MUT`
 Fusions | `FUSION` All fusions (note that many studies lack fusion data) | `FUSION`
-Copy Number Alterations | `AMP` Amplifications <br> `HOMDEL` Deep Deletions <br> `GAIN` Gains <br> `HETLOSS` Shallow Deletions <br> Comparison operators can also be used with `CNA` (e.g. `CNA >= GAIN` is the same as `AMP GAIN`) | `AMP` <br> `HOMDEL`
+Copy Number Alterations | `AMP` Amplifications <br> `DeepDel` Deep Deletions <br> `GAIN` Gains <br> `ShallowDel` Shallow Deletions <br> Comparison operators can also be used with `CNA` (e.g. `CNA >= GAIN` is the same as `AMP GAIN`) | `AMP` <br> `DeepDel`
```

Comment:
isn't it confusing to have different case for AMP/GAIN and DeepDel/ShallowDel? @jjgao